types/node/crypto.d.ts

```diff
@@ -3146,7 +3153,7 @@ declare module 'crypto' {
          * @since v15.6.0
          * @return Returns `email` if the certificate matches, `undefined` if it does not.
          */
-        checkEmail(email: string, options?: X509CheckOptions): string | undefined;
+        checkEmail(email: string, options?: Pick<X509CheckOptions, 'subject'>): string | undefined;
```

Comment:
```suggestion
        checkEmail(email: string, options?: Pick<X509CheckOptions, 'subject'>): 'email'| undefined;
```
can you amend the 'subject' prop as well to list `'default', 'always', or 'never'` union, please?

Reply:
that one, if changed, should be changed in v16 as well

Reply:
- `subject` already has the union `always` or `never`. `default` is not a valid option
- `email` is not a valid return type, the docs say `email` not `'email'`. See [tests here](https://github.com/nodejs/node/blob/e1fb6ae02ff01a0d29c95e4700c81d05e09f188d/test/parallel/test-crypto-x509.js#L133) for proof

Reply:
Sorry, I misread, you're correct!